tools/Config.mk

```diff
@@ -589,7 +589,7 @@ define CLEAN
 endef
 else
 define CLEAN
-	$(Q) rm -f *$(OBJEXT) *$(LIBEXT) *~ .*.swp $(OBJS) $(BIN) $(BIN).lock $(EXTRA)
+	$(Q) rm -f *$(OBJEXT) *$(LIBEXT) *~ .*.swp $(AROBJS) $(OBJS) $(BIN) $(BIN).lock $(EXTRA)
```

Comment:
if AROBJS just use for application build, please move this change into rule of application clean

Reply:
> if AROBJS just use for application build, please move this change into rule of application clean

DONE
in https://github.com/apache/nuttx-apps/pull/2138
closed this PR with duplicate modification https://github.com/apache/nuttx/pull/11007